Picker/ViewController.swift

```diff
@@ -8,33 +8,50 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
 
     @IBOutlet weak var collectionView: UICollectionView!
     
     override func viewDidLoad() {
-//        super.viewDidLoad()
+        super.viewDidLoad()
 //        let cardLayout = UICollectionViewFlowLayout()
 //        cardLayout.sectionInset = UIEdgeInsets(top: 30, left: 20, bottom: 50, right:20)
 //        cardLayout.itemSize = CGSize(width :236,height :367)
 //        cardLayout.minimumLineSpacing = 75
 //        cardLayout.scrollDirection = UICollectionViewScrollDirection.Horizontal
 //        collectionView = UICollectionView(frame: self.view.frame, collectionViewLayout: cardLayout)
-//        self.collectionView.dataSource = self
-//        self.collectionView.delegate = self
-//        self.collectionView.pagingEnabled = true
+        self.collectionView.dataSource = self
+        self.collectionView.delegate = self
+        self.collectionView.pagingEnabled = true
 //        collectionView.registerClass(CustomCellView.self, forCellWithReuseIdentifier: CustomCellView.cellIdentifier)
-//        collectionView.backgroundColor = UIColor.whiteColor()
-//        self.collectionView.showsHorizontalScrollIndicator = false
-//        self.view.addSubview(collectionView!)
+        collectionView.backgroundColor = UIColor.whiteColor()
+        self.collectionView.showsHorizontalScrollIndicator = false
+        self.view.addSubview(collectionView!)
     }
     
-//    func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
-//        var cell = collectionView.dequeueReusableCellWithReuseIdentifier(CustomCellView.cellIdentifier, forIndexPath: indexPath) as! CustomCellView
+
+    
+    func collectionView(collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 5
+    }
+    
+    func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
+        var cell = collectionView.dequeueReusableCellWithReuseIdentifier(CustomCellView.cellIdentifier, forIndexPath: indexPath) as! CustomCellView
 //        cell.backgroundColor = UIColor.blackColor()
-//        cell.imageView.image = NSURL(string: <#String#>)
-//        return cell
-//    }
+        let url = NSURL(string: "http://manage.allheart.com/images/detail/gg2015/gg-2015---.jpg")
+        let data = NSData(contentsOfURL: url!)
+        cell.imageView.image = UIImage(data: data!)
```

Comment:
There is a very cool image loading library [SDWebImage] (https://github.com/rs/SDWebImage). It automatically caches and loads images asynchronously... We should use that here.